packages/patternfly-4/content/get-started/gettingstarted-design.md

```diff
@@ -6,7 +6,29 @@ path: "/get-started/designers"
 
 <p>Before you begin, check out <a href="/get-started/about">About PatternFly 4</a> to get acquainted with the basic elements of the design system.</p>
 
-<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Designing with PatternFly 4</h1>
+<h1 class="pf-c-title pf-m-3xl pf-u-mt-xl">Get set up</h1>
+
+<h1 class="pf-c-title pf-m-2xl pf-u-mt-xl">Download and install the PatterFly 4 design kit</h1>
+
+<p>The design kit is a collection of assets that make it easy for you to create high-fidelity design mockups using PatternFly 4 components.</p>
+
+<p>The design kit works with Sketch libraries and templates. To use it, you'll need to have <div class="link"><a href="https://www.sketch.com/">Sketch</a> version 47 or above.</p>
+
+<h1 class="pf-c-title pf-m-xl pf-u-mt-xl">Download the design kit files:</h1>
+<p><b>Add the Sketch library</b></p>
+<ol>
+<li>Access the <div class="link"><a href="https://sketch.cloud/s/wxoZ9">PatternFly 4 design library</a> on Sketch cloud.</li>
+<li>Click <b>Add library to Sketch</b> on the bottom right to add the library. You'll be prompted with a modal. Click <b>Add library to Sketch</b> again. Sketch will show you the library in your preferences window. You're all set!</li>
+
+
+<p><b>Add the Sketch template</b></p>
+<ol>
```

Comment:
you have to close the previous <ol> before you open another one unless you intended this to be nested (then it should be inside an <li>